lib/health_web/templates/layout/_navbar.html.eex

```diff
@@ -37,10 +36,10 @@
       </li>
       <% else %>
       <li class="nav-item">
-        <%= link "Register", to: Routes.pow_registration_path(@conn, :new), class: "nav-link" %>
```

Comment:
Shouldn't this be nav_link? Or are you hoping to make that change later?

Reply:
This should ideally be `nav_link` or I can make the change later